functional_tests.go

```diff
@@ -10183,7 +10183,7 @@ func main() {
 		testGetObjectReadSeekFunctional()
 		testGetObjectReadAtFunctional()
 		testGetObjectReadAtWhenEOFWasReached()
-		testPresignedPostPolicy()
+		//testPresignedPostPolicy()
```

Comment:
looks like the test is disabled here @sinhaashish

Reply:
`Note : testPresignedPostPolicy is commented as it depends on minio PR minio/minio#8623`